src/pages/OrgRegister.tsx

```diff
@@ -1,56 +1,101 @@
-import React from 'react';
-import { Formik, Form, Field, ErrorMessage } from 'formik';
-import Footer from '../components/Footer';
-import Header from '../components/Header';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Logo from '../assets/logo.svg';
+import { IoPerson, IoMail, IoCreate } from 'react-icons/io5'
 
 export default function OrgRegister() {
+  const [name, setName] = useState("")
+
+  const handleChange = (value: string) => {
+    setName(value)
+  }
   return (
-    <>
-      <div className="bg-light-bg dark:bg-[#262E3D] h-screen flex flex-col">
-        <div className="container w-[95%] lg:w-[60%] mx-auto mt-16 flex-1 flex flex-col items-center justify-center px-2">
-          <div className="bg-white dark:bg-[#1F2A37] px-10 py-16 md:px-16 md:py-12 rounded text-black w-full">
-            <h1 className="mb-8 text-2xl sm:text-4xl text-center dark:text-white">
-              Register an organization
+    <div className="bg-light-bg dark:bg-[#262E3D] lg:h-full grow flex flex-col w-full">
+      <div className="flex flex-row items-center justify-center grow w-full">
+        <div className='hidden bg-[#1F2A37] dark:bg-[#262E3D] px-8 py-4 h-screen lg:w-[45%] lg:flex lg:flex-col grow pt-24'>
+          <div className='flex items-center justify-center text-white pt-16 pb-24'>
+            <img
+              className=" w-[40%] lg:px-5 cursor-pointer"
+              src={Logo}
```

Comment:
I think as we have a logo on the header, It will be better if we remove this logo.